### PR TITLE
SES 1.6.0 blog

### PR DESCRIPTION
**Pull request by kriskowal**
This adds a blog and kicks it off with release notes and migration instructions for SES 1.6.0.

### CODE REVIEW

**Comment by kriskowal** [comment]
attn @phoddie to show “how to blog” and for a hand shake on “whether to blog”.

File: src/content/docs/blog/ses-1.6.0.md [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+// Note the destructuring of namespace:
+const { namespace } = await compartment.import('my-module-specifier');
+console.log(namespace.default); // 42
+```
+
+The `import` method should simply return a promise for the module namespace
+object, just like dynamic `import` and XS.
+
+Version 1.6.0 introduces a `__noNamespaceBox__` option that makes the eventual
+correct behavior available today.
```
Comment (kriskowal) [inline src/content/docs/blog/ses-1.6.0.md:81]:
```suggestion
Version 1.6.0 introduces a `__noNamespaceBox__` option that makes the compatible and consistent behavior available today.
```

File: src/content/docs/blog/ses-1.6.0.md [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+  static module record, or virtual module source could be used in place of
+  a module descriptor.
+  These should now be boxed in a module descriptor with a `source` property.
+  (Relatedly, we have renamed the `@endo/static-module-record` package to
+  `@endo/module-source` to better reflect the direction of movement in the
+  ECMAScript standardization process.)
+
+  ```js
+  // before:
+  import { StaticModuleRecord} from '@endo/static-module-record';
```
Comment (gibson042) [inline src/content/docs/blog/ses-1.6.0.md]: Formatting doesn't look quite right in this block and the next one. 
```suggestion
  import { StaticModuleRecord } from '@endo/static-module-record';
```

File: src/content/docs/blog/ses-1.6.0.md [hunk 3]
```diff
@@ -0,0 +1,295 @@
...
+  const compartment = new Compartment(
+    {},
+    {
+      'my-module-specifier': new StaticModuleRecord(`
+      export default 42;
```
Comment (gibson042) [inline src/content/docs/blog/ses-1.6.0.md]: Also here (indentation).

File: src/content/docs/blog/ses-1.6.0.md [hunk 4]
```diff
@@ -0,0 +1,295 @@
...
+    {
+      'submodule/dependency': new StaticModuleRecord(`
+        export default 42;
+      `),
+      dependant: {
+        record: new StaticModuleRecord(`
+          import meaning from 'dependency';
+          export default meaning;
+        `),
+        specifier: 'submodule/depenant',
```
Comment (gibson042) [inline src/content/docs/blog/ses-1.6.0.md]: typo?

File: src/content/docs/blog/ses-1.6.0.md [hunk 5]
```diff
@@ -0,0 +1,295 @@
...
+        path.push(...importSpecifier.split('/'));
+        return path.join('/');
+      },
+    },
+  );
+  console.log(compartment.importNow('dependant').default); // 42
+  ```
+
+  ```js
+  // after
```
Comment (gibson042) [inline src/content/docs/blog/ses-1.6.0.md:258]: I'm not seeing a relevant difference between this block and its predecessor... what am I missing?

**Review by gibson042 — COMMENTED** [review]
I love the before/after blocks; very informative.

**Review by gibson042 — APPROVED** [review]

**Pull request merged**